src/devices/src/virtio/block/device.rs

```diff
@@ -427,6 +427,11 @@ pub(crate) mod tests {
     use utils::tempfile::TempFile;
     use vm_memory::GuestAddress;
 
+    use crate::virtio::block::test_utils::{
```

Comment:
nit: You could just use
`use crate:;virtio:;block::test_utils::*`. since we expect to define lots of stuff in these type of modules and use all of them

Reply:
https://github.com/firecracker-microvm/firecracker/pull/2151#discussion_r499448695 I was considering this when imported utilitary functions. Personally I consider that a good practice, let me know what you think.

Reply:
It is up to you.